packages/dashboard-v2/package.json

```diff
@@ -21,6 +21,7 @@
   "dependencies": {
     "@fontsource/sora": "^4.5.3",
     "@fontsource/source-sans-pro": "^4.5.3",
+    "byte-size": "^8.1.0",
```

Comment:
we're using https://www.npmjs.com/package/pretty-bytes in other repositories, would be nice to keep it consistent

Reply:
Fair enough 👍  aaaa9c13